platform-emf/src/main/java/com/softicar/platform/emf/form/factory/IEmfFormFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.softicar.platform.emf.form.factory;
+
+import com.softicar.platform.emf.form.IEmfForm;
+import com.softicar.platform.emf.form.IEmfFormFrame;
+import com.softicar.platform.emf.table.row.IEmfTableRow;
+
+public interface IEmfFormFactory<R extends IEmfTableRow<R, ?>> {
+
+	IEmfForm<R> getForm(IEmfFormFrame<R> frame, R tableRow);
```

Comment:
i think it should be rather called `createForm` to indicate it is a new form instance and that ownership is transfered